packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -108,10 +108,12 @@ export const firmwareUpdate = () => async (dispatch: Dispatch, getState: GetStat
     }
 
     // model 1
-    // ask user to unplug device (see firmwareMiddleware)
+    // ask user to unplug device if BL < 1.10.0 (see firmwareMiddleware), BL starting with 1.10.0 will automatically restart itself just like on model T
     // model 2 without pin
     // ask user to wait until device reboots
-    dispatch(setStatus(model === 1 ? 'unplug' : 'wait-for-reboot'));
+    dispatch(
+        setStatus(model === 1 && device.features.minor_version < 10 ? 'unplug' : 'wait-for-reboot'),
```

Comment:
A small nitpick. What if we used `compare-versions` package here? (I used in the new Onboarding as well https://github.com/trezor/trezor-suite/pull/3677/commits/c14a661ef47f9a08b3f5e5708b3d41bcc1903d61).

It'll be a more robust and more readable solution (+ handling possible bump of `device.features.major_version` as well).

Reply:
I would rather keep it as simple condition, `compare-version` seems to me like an overkill for such a simple task. What do you mean by handling possible bump of major_version? This should be really bullet proof as with fw newer than 1.10.0 we will always want to wait for a reboot and newer version will always have minor_version set to at least 10.